vignettes/bisg.Rmd

```diff
@@ -3,53 +3,80 @@ title: "Bayesian Improved Surname Geocoding (BISG)"
 
 ## *Performing BISG on your data*
-The first step in performing BISG is to geocode your voter file addresses. For information on geocoding, visit the Geocoding Vignette. 
+We will perform BISG using the previous Gwinnett and Fulton county voter registration data called `ga_geo.csv` that was geocoded in the **eiCompare: Geocoding vignette**. 
 
-In this tutorial, we will be using the East Ramapo, New York voter file called, **voter_file_geocoded**, that consists of indivduals registering to vote during the year of 2016. 
+The first step in performing BISG is to geocode your voter file addresses. For information on geocoding, visit the Geocoding Vignette. 
 
 Let's begin by loading your geocoded voter data into R/RStudio.
 
 ### Step 1: Load R libraries/packages, voter file, and census data
-
 Load the R packages needed to perform BISG. If you have not already downloaded the following packages, please install these packages.
+
 ```{r}
-# Load libraries/packages
-
-suppressMessages(c(
-  library(devtools),
-  install.packages("tidyverse"),
-  library(tidyverse),
-  install.packages("stringr"),
-  library(stringr),
-  install.packages("sf"),
-  library(sf),
-  install.packages("eiCompare"),
-  library(eiCompare),
-  library(wru),
+# Install eiCompare
+suppressPackageStartupMessages({
+  devtools::install_github("RPVote/eiCompare@master")
+})
+```
+
+```{r}
+# Load libraries
+suppressPackageStartupMessages({
+  library(devtools)
+  library(tidyverse)
+  library(stringr)
+  library(tigris)
+  library(leaflet)
+  library(sf)
+  library(eiCompare)
+  library(wru)
   library(readr)
-))
+})
+```
+
+```{r}
+# source files
+source("~/github/eiCompare/R/wru_predict_race_wrapper.R")
+source("~/github/eiCompare/R/voter_file_utils.R")
+```
+
+Load in census data, the shape_file and geocoded voter registation data with latitude and longitude coordinates Gwinnett and Fulton .
+
+Make sure to load your census data that details certain geographies (i.e. counties, cities, tracts, blocks, etc.)
+```{r}
+# Load Georgia census data
+path_census <- "~/shared/georgia/"
+census_data <- readRDS(paste(path_census, "georgia_census.rds", sep = ""))
+```
+
+Next, load the state shape file using the sf::st_read() function.
+```{r, echo=FALSE}
+# Load Georgia block shape file
+shape_file <- blocks(state = "GA", county = c("Gwinnett", "Fulton"))
 ```
 
-Load in East Ramapo, NY School District voter registation data.
+Load geocoded voter file.
 ```{r}
 # Load geocoded voter registration file
-voter_file_geocoded <- read_csv("~/shared/east_ramapo/data/ram17_clean.csv")
+voter_file_geocoded <- read_csv("~/github/eiCompare/data/ga_geo.csv")
```

Comment:
same comment as above - this directory won't be available to a vignette user

Reply:
I can load the file onto github. It comes from a toy dataset that the user makes within the geocoding vignette.